src/main/java/players/basicMCTS/BasicMCTSParams.java

```diff
@@ -0,0 +1,58 @@
+package players.basicMCTS;
+
+import core.AbstractGameState;
+import core.interfaces.IStateHeuristic;
+import players.PlayerParameters;
+
+import java.util.Arrays;
+
+
+public class BasicMCTSParams extends PlayerParameters {
+
+    public double K = Math.sqrt(2);
+    public int rolloutLength = 10; // assuming we have a good heuristic
+    public int maxTreeDepth = 1000; // effectively no limit
+    public double epsilon = 1e-6;
+    public IStateHeuristic heuristic = AbstractGameState::getHeuristicScore;
+
+    public BasicMCTSParams() {
+        this(System.currentTimeMillis());
+    }
+
+    public BasicMCTSParams(long seed) {
+        super(seed);
+        addTunableParameter("K", Math.sqrt(2), Arrays.asList(0.0, 0.1, 1.0, Math.sqrt(2), 3.0, 10.0));
+        addTunableParameter("rolloutLength", 10, Arrays.asList(0, 3, 10, 30, 100));
+        addTunableParameter("maxTreeDepth", 10, Arrays.asList(1, 3, 10, 30, 100));
```

Comment:
I strongly recommend ensuring that the default in addTunableParameter matches the value in the main class. So set maxTreeDepth to 1000 in both.

This has caught me out before (and caused me to have to rerun 6 weeks worth of experiments becasue the default you see at first glance is not the one actually used if you fail to specify it in the json)